tmt/export.py

```diff
@@ -749,17 +750,20 @@ def export_to_polarion(test: 'tmt.Test') -> None:
     # List of bugs test verifies
     bug_ids = []
     requirements = []
-    for link in test.link:
-        bug_ids_search = re.search(RE_BUGZILLA_URL, link['verifies'])
-        if bug_ids_search:
-            bug_ids.append(int(bug_ids_search.group(1)))
-        else:
-            log.debug('Failed to find bug ID in verifies link')
-        polarion_url_search = re.search(RE_POLARION_URL, link['verifies'])
-        if polarion_url_search:
-            requirements.append(polarion_url_search.group(1))
-        else:
-            log.debug('Failed to find Polarion URL in verifies link')
+    for link in test.link.get():
+        try:
+            bug_ids_search = re.search(RE_BUGZILLA_URL, link['verifies'])
+            if bug_ids_search:
+                bug_ids.append(int(bug_ids_search.group(1)))
+            else:
+                log.debug('Failed to find bug ID in verifies link')
+            polarion_url_search = re.search(RE_POLARION_URL, link['verifies'])
+            if polarion_url_search:
+                requirements.append(polarion_url_search.group(1))
+            else:
+                log.debug('Failed to find Polarion URL in verifies link')
+        except Exception as err:
+            log.debug(err)
```

Comment:
Really an silent error? Shouldn't be user notified?

Reply:
this is generally an error like link doesn't have attribute verified, because most links are relates and have no influence on anything

Reply:
and also this is just returning what was removed by mistake

Reply:
Make sense. Once it is too noisy for user we can change